### PR TITLE
[WM-1992] add statement to README.md and DEVELOPMENT.md about including Jira IDs in PR titles

### PR DESCRIPTION
**Pull request by mspector**

### CODE REVIEW

**Review by kpierre13 — APPROVED** [review]

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 TODO: Complete me! Should be of similar quality to the ECM documentation.
 
+## Contributing
+
+For compliance reasons, all pull requests **must** be submitted with a Jira ID in the pull request title.
+Include the Jira ID at the beginning of the title, enclosed in square brackets.
```
Comment (salonishah11) [inline DEVELOPMENT.md:8]: small nit: maybe we can remove the requirement of "enclosed in square brackets". I don't think that's mandatory no?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Composite Batch Analysis Service (C-BAS)
 
+## Contributing
+
+For compliance reasons, all pull requests **must** be submitted with a Jira ID in the pull request title.
+Include the Jira ID at the beginning of the title, enclosed in square brackets.
```
Comment (salonishah11) [inline README.md:6]: Same comment here

File: DEVELOPMENT.md (hunk 1 above)
Comment (cjllanwarne) [reply DEVELOPMENT.md:8]: Yeah I never put them in square brackets 😆

Comment (cjllanwarne) [reply DEVELOPMENT.md:8]: I'd also say, if there is more than one relevant ticket, include them all, eg `WM-1997, WM-2002, WM-2005: Fix for many bugs with the same root cause`, etc.

**Review by cjllanwarne — APPROVED** [review]
LGTM, with a few minor suggestions

**Pull request merged**